NoMansLand/Assets/Scripts/SpaceshipManager.cs

```diff
@@ -117,7 +117,14 @@ public static void addBonusTime()
 			bonusTime += TIME_BONUS;
 		}
 		// DropPartFromShip adds the dropped part to the list and makes it drop
-		public static void DropPartFromShip(string partName) {
+		public static void DropPartFromShip(string partName)
+		{
+			if (partName.Contains("(Clone)"))
```

Comment:
Could this possibly overlap and run for HealthItem clones and other collectible clones which can be picked up?

Reply:
That should be fine, since this doesn't touch picking up items, just dropping them from the ship. This if statement should never be triggered in the first place, as it is only those pieces with DisableIsKinematic enabled that can even possibly DropPartFromShip. So the healthitems should be fine; for the collectible clones that can be picked up they have a drop time set to beyond the game ending so it should be fine also